<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #181818;
text-align: center;
font-size: 32px;
font-family: Poppins;
font-style: normal;
font-weight: 900;
line-height: normal;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/logosentiment.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblEmotion">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>90</horstretch>
      <verstretch>90</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/Smiling.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblText">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>250</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #000;
font-size: 12px;
font-family: Poppins;
font-style: normal;
font-weight: 300;
line-height: normal;</string>
    </property>
    <property name="text">
     <string>???</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputText">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>310</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>310</width>
      <height>41</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #000;
font-size: 12px;
font-family: Poppins;
font-style: normal;
font-weight: 400;
line-height: normal;
padding:5px;
border-radius: 5px;
border: 1px solid #EBEBEB;
background: #FFF;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Enter text to analyse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAnalyze">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>370</y>
      <width>164</width>
      <height>38</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>164</width>
      <height>38</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnAnalyze{
border-radius: 3px;
background: #36B0F4;
color: #FFF;
font-size: 12px;
font-family: Poppins;
font-style: normal;
font-weight: 300;
line-height: normal;
}
#btnAnalyze:hover{
border-radius: 3px;
background: #029AEF;
}
</string>
    </property>
    <property name="text">
     <string>Analyze</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorMessage">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>280</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #E93E30;
font-size: 10px;
font-family: Poppins;
font-style: normal;
font-weight: 300;
line-height: normal;
padding: 5px;</string>
    </property>
    <property name="text">
     <string>ERROR</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
